views/js/controller/runner/runner.js

```diff
@@ -44,7 +46,8 @@ define([
      */
 
     runner.registerProvider('qti', qtiProvider);
-    proxy.registerProxy('qtiServiceProxy', qtiServiceProxy);
+    proxy.registerProvider('qtiServiceProxy', qtiServiceProxy);
+    communicator.registerProvider('poll', pollProvider);
```

Comment:
In fact, the type of communication is set from a config entry. Here it is just the provider registration.